rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -438,7 +448,7 @@ struct nt_p2p_port
 	shared_mutex s_sign_mutex;
 	std::vector<std::pair<std::pair<u32, u16>, std::vector<u8>>> sign_msgs{};
 
-	std::array<u8, 65535> p2p_recv_data{};
+	std::array<u8, 64*1024> p2p_recv_data{};
```

Comment:
Max udp packet size is 65535 bytes, not 65536 bytes...

Reply:
Yep, my bad google-foo which just seemed to result in lots of references to 'maximum UDP packet size is 64kB'... I'll pull this change out.